### PR TITLE
feat: adds new token types

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Adding new token types to the list

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: src/BasisTheory.net/Tokens/Constants/TokenTypes.cs [hunk 1]
```diff
@@ -5,5 +5,9 @@ public class TokenTypes
         public const string Token = "token";
```
Comment (jleon15) [inline src/BasisTheory.net/Tokens/Constants/TokenTypes.cs:5]: why is this not an enum?

Comment (bweber) [reply src/BasisTheory.net/Tokens/Constants/TokenTypes.cs:5]: C# doesn't support string enums. You have to use something like Humanizer to make that work.

Comment (jleon15) [reply src/BasisTheory.net/Tokens/Constants/TokenTypes.cs:5]: gotcha, this is just used by the faker in the tests, i guess it's just to expose the list to consumers of the SDK. We don't want to force them to use Humanizer then?

**Comment by koalaty-code** [comment]
@jleon15 please ensure the merge commit message is prefixed with the appropriate semantic release verbiage to properly version the new release. E.g. `fix: ` to bump patch version or `feat: ` to bump minor version.

**Review by koalaty-code — APPROVED** [review]

**Pull request merged**